src/collect-lp.ts

```diff
@@ -207,9 +219,9 @@ export class LpCollector {
             quoteToWithdraw
           );
         }
-
-        // FIXME: this calculates how much we LP we tried to redeem, not how much we actually redeemed
-        return wdiv(quoteToWithdraw, exchangeRate);
+        
+        // Return the actual LP used
+        return actualLpUsed;
```

Comment:
Could just `return lpBalanceBefore.sub(lpBalanceAfter)` rather than creating a new variable for it on line 209.

Reply:
agree